src/producer.ts

```diff
@@ -128,6 +176,43 @@ export class Producer {
         }
     }
 
+    private async _multiStationProduce(
+        {
+            message,
+            ackWaitSec = 15,
+            asyncProduce = true,
+            headers = new MsgHeaders(),
+            msgId = null,
+            producerPartitionKey = null,
+            producerPartitionNumber = -1
+        }:
+            {
+                message: any;
+                ackWaitSec?: number;
+                asyncProduce?: boolean;
+                headers?: any;
+                msgId?: string;
+                producerPartitionKey?: string;
+                producerPartitionNumber?: number;
+            }
+    ): Promise<void> {
+        const tasks = this.stationNames.map(stationName => this.connection.produce(
```

Comment:
make it work consecutively and return an error to the user if the produce failes